protocol-spec/README.md

```diff
@@ -1,6 +1,6 @@
 # The Buttplug Sex Toy Control Standard
 
-* **Version:** 0.1.0
+* **Version:** 0.2.0
```

Comment:
Mostly because I didn't want to mess up the versioning scheme for the overall project. The Schema also has a 0.2.0 version in it, but that's closer related to the release than the message version number.

Reply:
You know that buttplug-js is currently sitting at something like 0.4.1, right? :)

Reply:
Yeah. I expressed concern back at 0.2.0 of the npm... But this gets around it well.